rules-reviewed/eap7/eap71/eap72-picketlink-STS-token-format-change.windup.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="eap72-picketlink-STS-token-format-change"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://windup.jboss.org/schema/jboss-ruleset http://windup.jboss.org/schema/jboss-ruleset/windup-jboss-ruleset.xsd">
+    <metadata>
+        <description>
+            This ruleset provides analysis of applications that Picketlink STS and may require
+            individual attention when migrating to JBoss EAP 7.2.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,3.0.0.Final" />
+            <addon id="org.jboss.windup.rules,windup-rules-java,3.0.0.Final" />
+        </dependencies>
+        <sourceTechnology id="eap" versionRange="(,7.1]" />
+        <targetTechnology id="eap" versionRange="[7.2,)" />
```

Comment:
the target versionRange needs to be [7,8) because we want this rule to fire if someone specifies a target of eap:7 and unfortunately the rules engine doesn't recognise minor versions as being part of the same overall version